dids/didjwk.go

```diff
@@ -74,8 +74,10 @@ func NewDIDJWK(opts ...NewDIDJWKOption) (BearerDID, error) {
 	return did, nil
 }
 
+type JWKResolver struct{}
```

Comment:
this part is meh. the struct doesn't do much. but it cleans up the interface for resolving dids

Reply:
it's a bit strange to me to resolve a "JWK" tho... semantically speaking, that doesn't add up to what the concept of a JWK is, no?

I get this is within the context of DIDs, but still maybe slightly conflating

Reply:
agreed @KendallWeihe . @mihai-chiorean and i just discussed a refactor where each did method has it's own package which i'm working on right now so we decided to merge this in knowing that the confusion goes away in a subsequent PR